xrally_kubernetes/service.py

```diff
@@ -78,3 +154,75 @@ def list_namespaces(self):
                  "uid": r.metadata.uid,
                  "labels": r.metadata.labels}
                 for r in self.v1_client.list_namespace().items]
+
+    @atomic.action_timer("kube.create_namespace")
+    def create_namespace(self, name):
+        """Create namespace.
+
+        :param name: namespace name
+        """
+        manifest = {
+            "apiVersion": "v1",
+            "kind": "Namespace",
+            "metadata": {
+                "name": name,
+                "labels": {
+                    "role": "test-%s" % name
```

Comment:
let's do not use "test-" prefix. It will be a real issue while designing cleanup.